src/Lang/ne.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Date Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used by the date library. Each line can
+    | have a singular and plural translation separated by a '|'.
+    |
+    */
+
+    'ago'       => ':time पहिले',
+    'from_now'  => ':time देखी',
+    'after'     => ':time पछि',
+    'before'    => ':time अघि',
+    'year'      => ':count वर्ष',
+    'month'     => ':count महिना',
+    'week'      => ':count हप्ता',
+    'day'       => ':count दिन',
+    'hour'      => ':count घण्टा',
+    'minute'    => ':count मिनेट',
+    'second'    => ':count सेकेण्ड',
+
+    'january'   => 'जेनवरी',
```

Comment:
जनवरी

Reply:
I transliterated the months because we don't have a true representation of those month names in Nepali.  Thus, I followed the good folks of Wikipedia and used the transliteration they were using: 

Source: https://ne.wikipedia.org/wiki/%E0%A4%9C%E0%A4%A8%E0%A4%B5%E0%A4%B0%E0%A5%80 

If there's a better source I can take reference from, I'd love to do that.

Reply:
roger that. my mistake